modules/openapi-generator/src/main/resources/python/partial_api.mustache

```diff
@@ -44,11 +44,9 @@
         )
 
         _response_types_map: Dict[str, Optional[str]] = {
-            {{#returnType}}
             {{#responses}}
             {{^isWildcard}}
             '{{code}}': {{#dataType}}"{{.}}"{{/dataType}}{{^dataType}}None{{/dataType}},
             {{/isWildcard}}
             {{/responses}}
-            {{/returnType}}
```

Comment:
What was the purpose of the `{{#returnType}}` condition here?

Reply:
This does not seem to affect the operation.
The `{{#returnType}}` simplifies the generated source code, but I don't think the simplification is necessary.

Reply:
Awesome, thanks for checking!